doc/sphinx/conf.py

```diff
@@ -213,6 +213,7 @@ def setup(app):
     'versions': [
         ("dev", "https://coq.inria.fr/doc/master/refman/"),
         ("stable", "https://coq.inria.fr/refman/"),
+        ("9.0", "https://coq.inria.fr/doc/v9.0/refman/"),
         ("8.20", "https://coq.inria.fr/doc/v8.20/refman/"),
         ("8.19", "https://coq.inria.fr/doc/v8.19/refman/"),
         ("8.18", "https://coq.inria.fr/doc/v8.18/refman/"),
```

Comment:
Can you at the same time change these links to not be targeting a branch, but the latest tag?

Reply:
This seems to result in a redirect to the same pages, though. Or I misunderstood the shape of the expected links.

Reply:
Concretely what I proposed:
```python
        ("8.19", "https://coq.inria.fr/doc/V8.19.2/refman/"),
        ("8.18", "https://coq.inria.fr/doc/V8.19.2/refman/"),
```
This page do not redirect.